server/integration/feature-pack/src/main/resources/content/bin/standalone.sh

```diff
@@ -333,7 +333,7 @@ if [ "$SECMGR" = "true" ]; then
 fi
 
 if [ "x$USE_JMX_COLLECTORS" != "x" ]; then
-    . $DIRNAME/jolokia.sh $JMX_CONF
+    . $DIRNAME/prometheus.sh $JMX_CONF
```

Comment:
You forgot to `git add prometheus.sh` somewhere

Reply:
Indeed. Thanks @tristantarrant